tests/testutils/kubeutils/kind_cluster.go

```diff
@@ -45,7 +46,13 @@ const kindConfigTemplate = `kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
-    image: kindest/node:v1.26.0
```

Comment:
I've been planning on expanding this to allow specifying arbitrary nodes (to allow specifying multiple control-plane and workers). I can envision cases where we use different images for different instances so would need a mechanism of specifying this field. Did you find it necessary to remove this or can it be templated instead?

Reply:
yes, instead of having this in the config yaml, I made a change to make this a parameter of the cluster config, so it can be set. The kind command takes an `--image` parameter so it can be set separately from the kind config.

Reply:
my use case right now is to use the same kind config with different images. Maybe there's more subtlety down the road for image support.

Reply:
Do you templating here instead if it wasn't a necessary change (as in the value is only respected if using the cli option)? Seems unnecessary to add that approach and then to require shifting back in the future.

Reply:
Lost the thread on this being resolved here but since this is a node setting I think it should be templated instead of moving to the cli option.

Reply:
Oh sorry I thought I answered I don't mind either way. Let me see to it.